js/ui/calendar/ui.calendar.base_view.js

```diff
@@ -198,12 +205,29 @@ const BaseView = Widget.inherit({
                 });
             }
         }));
+
```

Comment:
I undarstand that you don't create _renderEvents() function, but it name seems inappropriate. I would expect something like `initActions` or `createActions` here. What do you think to change this function name? In any case it a note only.

Reply:
I agree that name `renderEvents` sounds pretty inappropriate, but I checked our other components and it seems like we use `initActions` function only for creating actions and `renderEvents` when we also add some events listeners to elements. This function name is common for other editors, so I prefer to keep this as it is
![image](https://user-images.githubusercontent.com/29150145/220602426-4a0f0a2e-03a0-45ca-af2d-7af3981021ea.png)

Reply:
I agree, this question is not for this PR